### ISSUE TITLE
MaterialSearBar opens Google Search

### ISSUE THREAD

**Issue by sijan8s3** [AbbasHassan/PlacesProject#2]
I'm using the map inside the fragment,
I implemented the map and material search correctly but when I click on the softkey search(keyboard search) button, It directly opens google search. Please help!

**Comment by imranhaider-se** [comment]
I'm also facing 
the same problem.Did you find any solution??

**Comment by sijan8s3** [comment]
> I'm also facing
> the same problem.Did you find any solution??

no, rather I used the Place Autocomplete widget of google map API,
but it charges much!

**Comment by imranhaider-se** [comment]
How much and can you snd me the source code?

**Comment by sijan8s3** [comment]
https://www.youtube.com/watch?v=nDC-frkH_LA
this will help you!

**Comment by imranhaider-se** [comment]
Thanks

**Comment by siddiqueji** [comment]
go to the following link, there is an option for kotlin code also

https://developers.google.com/places/android-sdk/autocomplete#maps_places_on_activity_result-java
